SupportBank/HelperFunctions.cs

```diff
@@ -33,15 +38,33 @@ public void ListAccount(string name)
             }
         }
 
-        public void ParseFileAndPopulateDictionary()
+        public void ParseFileAndPopulateDictionary(string filename)
         {
-            using var reader = new StreamReader("./Transactions2014.csv");
+            logger.Info($"Reading file {filename}");
+            using var reader = new StreamReader($"./{filename}");
             using var csv = new CsvReader(reader, CultureInfo.InvariantCulture);
             csv.Configuration.PrepareHeaderForMatch = (string header, int index) => header.ToLower();
+            var options = new TypeConverterOptions
+            {
+                DateTimeStyle = DateTimeStyles.None,
+                Formats = new[] {"dd/MM/yyyy"},
+            };
+            csv.Configuration.TypeConverterOptionsCache.AddOptions<DateTime>(options);
+            logger.Info("File read successfully - Beginning to add transactions");
```

Comment:
Minor but I think adding an empty line before the logger line above would make this easier to read 🙂